src/rest_parser.cpp

```diff
@@ -273,6 +294,19 @@ bool RestParser::setDimOrValidate(tensorflow::TensorProto& proto, int dim, int s
     }
 }
 
+bool getB64FromValue(const rapidjson::Value& value, std::string& b64Val) {
+    if (value.IsObject()) {
```

Comment:
Avoid multiple indention, this could be easier to read by checking each statement in separate lines:
```
if (!values.IsObject()) {
  return false;
}
if (!...) {
...
}
```